diff --git a/node_modules/@graphql-codegen/visitor-plugin-common/index.cjs.js b/node_modules/@graphql-codegen/visitor-plugin-common/index.cjs.js
index 52bb890..73ddd3b 100644
--- a/node_modules/@graphql-codegen/visitor-plugin-common/index.cjs.js
+++ b/node_modules/@graphql-codegen/visitor-plugin-common/index.cjs.js
@@ -2188,7 +2188,7 @@ class ClientSideBaseVisitor extends BaseVisitor {
         return '';
     }
     _prepareDocument(documentStr) {
-        return documentStr;
+        return graphql.stripIgnoredCharacters(documentStr);
     }
     _gql(node) {
         const fragments = this._transformFragments(node);
